source/localizable/tutorial/acceptance-test.md

```diff
@@ -0,0 +1,145 @@
+```
+
+Opening the new test file will reveal some boilerplate code that will try to go to the `list-rentals` route and verify that the route is loaded.
+This boiler plat code is there to guide you into your first working acceptance test.
+Since we are testing our index route, which is `/`, we'll replace occurances of `/list-rentals` with `/`.
+
+```javascript{-6,+7,-8,+9,-12,+13}
+import { test } from 'qunit';
+import moduleForAcceptance from 'super-rentals/tests/helpers/module-for-acceptance';
+
+moduleForAcceptance('Acceptance | list-rentals');
+
+test('visiting /list-rentals', function(assert) {
+test('visiting /', function(assert) {
+  visit('/list-rentals');
+  visit('/');
+
+  andThen(function() {
+    assert.equal(currentURL(), '/list-rentals');
+    assert.equal(currentURL(), '/');
+  });
+});
+```
+
+Now run your test suite with `ember t -s` from the command line and you'll see one successful test!
+
+As mentioned before, this test boilerplate is just for checking the environment, so now lets replace this test with our list of goals.
+
+```/tests/acceptance/list-rentals-test.js
+import { test } from 'qunit';
+import moduleForAcceptance from 'super-rentals/tests/helpers/module-for-acceptance';
+
+moduleForAcceptance('Acceptance | homepage');
+
+test('should list available rentals.', function (assert) {
+});
+
+test('should display some key information about each rental in the list.', function (assert) {
+});
+
+test('should link to information about the company.', function (assert) {
+});
+
+test('should link to contact information.', function (assert) {
+});
+
+test('should filter the list of rentals by city.', function (assert) {
+});
+```
+
+These tests will fail, since ember tests will always fail without assertions.
+Since we have an idea of what we want our application to look like, we can also add some details to the tests.
+
+Ember acceptance tests provide test helpers to perform common tasks, such as visiting routes, filling in fields, clicking on elements, and waiting on renders.
+
+To check that rentals are listed, we'll first visit the index route and check that the results contain 3 listings.
+
+```/tests/acceptance/list-rentals-test.js
+test('should list available rentals.', function (assert) {
+  visit('/');
+  andThen(function () {
+    assert.equal(this.$('.results .listing').length, 3);
+  });
+});
+```
+The test assumes that we show the list of rentals in an element with class `results`, and that each rental will have an element with class `listing`.
+
+The [`visit`](http://emberjs.com/api/classes/Ember.Test.html#method_visit) helper loads the route specifed for the given URL.
+
+The [`andThen`](https://guides.emberjs.com/v2.5.0/testing/acceptance/#toc_wait-helpers) helper waits for all previously called asynchronous test helpers to complete before executing its provided function.  
+In this case we need to wait for the page to be loaded after `visit` before asserting that the listings have been loaded onscreen.
```

Comment:
Can we say these two sentences in plainer non-tech English? ;-)  Happy to suggest a few sentences if you'd like ...